runtime/neumann/src/lib.rs

```diff
@@ -775,7 +794,7 @@ impl Contains<Call> for ClosedCallFilter {
 			Call::AutomationTime(_) => false,
 			Call::Balances(_) => false,
 			Call::Bounties(_) => false,
-			Call::CollatorSelection(_) => false,
+			Call::ParachainStaking(_) => false,
```

Comment:
if we close the parachain staking gate, what happens?

Reply:
I believe that this just shuts off access to the parachain_staking extrinsics for non sudo accounts. @rhuttman?

Reply:
correct